Backend/dev/migrations/dynamic-offer-driver-app/0209-added-stats-column-and-driver-hometown.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE atlas_driver_offer_bpp.driver_stats ADD COLUMN total_earnings int DEFAULT 0;
+ALTER TABLE atlas_driver_offer_bpp.driver_stats ADD COLUMN bonus_earned int DEFAULT 0;
+ALTER TABLE atlas_driver_offer_bpp.driver_stats ADD COLUMN late_night_trips int DEFAULT 0;
+ALTER TABLE atlas_driver_offer_bpp.driver_stats ADD COLUMN earnings_missed int DEFAULT 0;
+
+ALTER TABLE atlas_driver_offer_bpp.person ADD COLUMN hometown character varying(255);
+ALTER TABLE atlas_driver_offer_bpp.person ADD COLUMN languages_spoken text[] DEFAULT '{}'::text[];
```

Comment:
This isn't json, right?

Reply:
it's creating an empty list as the default value